src/main/kotlin/org/koitharu/kotatsu/parsers/site/MadaraParser.kt

```diff
@@ -288,8 +288,8 @@ internal abstract class MadaraParser(
 		override val tagPrefix = "genres/"
 	}
 
-	@MangaSourceParser("MANGAWEEBS", "MangaWeebs", "en")
-	class MangaWeebs(context: MangaLoaderContext) : MadaraParser(context, MangaSource.MANGAWEEBS, "mangaweebs.in")
```

Comment:
Why removed??

Reply:
Yeah, interesting question. Maybe in some countries ISPs blocking this site?

Reply:
Removing an existing source is not a good idea anyway. It will break manga in history/favorites, so, I`ll return it back. @OshekharO